### PR TITLE
bug fixes and improvements

### LINKED ISSUE #7: Add concise process section
We need a section that describes the process as succinctly as possible. Just a ol list of short steps... hopefully no more than 5.

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #7

I've redesigned the site to:

* Show how many active incubations there are (e.g., 120).
* include an short about section, make it again clear that these are not web standards.
* provide a description of each incubation (pulled from github), when available.
* show how long an incubation has been running for and last updated.
* How many Github stars an incubation has.
* Better description of the WICG. 
* Link to proposals. 
* Displays some basic statistics about incubations.

It looks like: 

Screenshot of site

And mobile:

Screenshot of site on mobile

WDYT?

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Oh, I also [set up the a GitHub action](https://github.com/WICG/wicg.io/blob/gh-pages/.github/workflows/generator.yml) to automatically regenerate the repos.json file every 24 hours.

File: index.html
Comment (yoavweiss) [inline index.html]: s/and the/that/?

**Review by yoavweiss — COMMENTED** [review]
Awesome!!

File: index.html
Comment (marcoscaceres) [reply index.html]: Oops, yep.

**Comment by yoavweiss** [comment]
Currently the site is served from the same server as Discourse, but I'm thinking that with the GH Actions changes, we can move it to GH pages.

**Comment by marcoscaceres** [comment]
I can’t recall which one of us actually hosts the website? Was it you or me?  😅

**Comment by yoavweiss** [comment]
Yeah, I'm managing the server 😄 

^^ @cwilso @LJWatson - thoughts on the revamp?

**Comment by marcoscaceres** [comment]
Happy to move to gh-pages. You’ll need to do the CNAME magic, but I recall it’s not very painful.

File: index.html
Comment (marcoscaceres) [inline index.html]:
```suggestion
          incubation of new ideas that could one day become web standards.
```

**Comment by marcoscaceres** [comment]
I've added some basic statistics about incubations. The "Active Incubations" section now reads, for example:

> There are 120 incubations actively being worked on by the WICG community. The [mean](https://en.wikipedia.org/wiki/Mean) incubation time for these projects is 3 years and 10 months and the [median](https://en.wikipedia.org/wiki/Median) incubation time is 3 years and 6 months. The longest running incubation is [serial](https://wicg.github.io/serial/), which has been incubating for approximately 9 years and 3 months. The shortest incubation is [capture-all-screens](https://github.com/WICG/capture-all-screens), which has been incubating for 1 month.
>
>Incubations are listed from most active participation to least active, based on the community's interaction with issues and incoming commits.

We might need to fix the wording a little bit so that that doesn't mean that incubations finish within 3+ years... just the on average they have been up and running for 3+ years.

**Comment by marcoscaceres** [comment]
I got some feedback from my colleagues at Apple and they asked if I could show what's been archived and when. The Github API is able to tell us if something is archived, but not exactly **when** it was archived. So, instead, I've used `pushed_at` as proxy for that. It's fairly accurate, as we usually update README.md of an incubation before archiving.

This is what it now looks like. I've made it clear that each is archived and included the median time for archived incubations. For example: 

> The following 58 incubations have been archived and are no longer actively being pursued by the WICG community. This indicates that either they have successfully progressed to the [W3C](https://www.w3.org/) or the [WHATWG](https://whatwg.org/), or they have been abandoned.

> The [median](https://en.wikipedia.org/wiki/Median) time for archived incubations is 1 year and 9 months. The shortest incubation before being archived was network-error-logging, which lasted for approximately less than a month. The longest running incubation before being archived was container-queries, which lasted for approximately 7 years and 9 months.

Screenshot of updated website showing the text above

**Comment by marcoscaceres** [comment]
To make it easier to view the changes, the site is available at https://marcoscaceres.github.io/wicg.io/

**Comment by yoavweiss** [comment]
Looks great! :) Let's wait for feedback from Leonie and Chris before landing.

**Comment by LJWatson** [comment]
Thanks @marcoscaceres. These changes LGTM.

I wonder if we should remove the Discourse link whilst we're at it?

**Comment by marcoscaceres** [comment]
Ok, cool. I've removed the link to Discourse.

File: index.html
Comment (cwilso) [inline index.html]: This should probably be clearer that they have progressed to "a REC track in a Working Group at the W3C" rather than just "W3C" - as is, it doesn't delineate enough between WICG (which is in W3C) and actual-standards-track-venue-in-W3C.

**Review by cwilso — APPROVED** [review]
Overall, LGTM.  Added one comment that you might want to address, but I don't think it's blocking.

File: index.html
Comment (marcoscaceres) [reply index.html]: Good point. Will fix.

Comment (marcoscaceres) [inline index.html]:
```suggestion
          progressed to the <a href="https://www.w3.org/2021/Process-20211102/#rec-track">recommendation track</a> in a <a href="https://www.w3.org/">W3C</a> <a href="https://www.w3.org/groups/wg/">Working Group</a> or the
```

**Pull request merged**
